global_costmap:

  map_type: costmap
  track_unknown_space: true
  unknown_cost_value: 255
  obstacle_range: 2.5
  raytrace_range: 3.0
  footprint: [[0.45, 0.25],
              [-0.45, 0.25],
              [-0.45, -0.25],
              [0.45, -0.25]]
  inflation_radius: 0.32
#   transform_tolerance: 0.5
  inscribed_radius: 0.14
  circumscribed_radius: 0.18

  global_frame: map
  robot_base_frame: base_link
  update_frequency: 2.0
  static_map: true
  publish_frequency: 1.0
#   raytrace_range: 30.0
#   obstacle_range: 5.0
#   rolling_window: false
  width: 60.0
  height: 60.0
  resolution: 0.05
  origin_x: -30.0
  origin_y: -30.0

#   TODO try changing parameter
  cost_scaling_factor: 10.0
